### ISSUE TITLE
develop branch on commit da5fcad fails unit tests

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-cli#319]
It works to install a installation package based on commit 1565fec. On to of that  PR "Unit test for PO files" (#314) has been merge to form the current develop branch (on commit da5fcad).

Installation package based on develop branch fails.

```
t/00-load.t ... 1/? # Testing Zonemaster::CLI v5.0.2, Perl 5.032001, /usr/local/bin/perl
t/00-load.t ... ok   
t/po-files.t .. usage: make [-BeikNnqrSstWwX]
            [-C directory] [-D variable] [-d flags] [-f makefile]
            [-I directory] [-J private] [-j max_jobs] [-m directory] [-T file]
            [-V variable] [-v variable] [variable=value] [target ...]

    #   Failed test 'make show-fuzzy exits with value 0'
    #   at t/po-files.t line 35.
    #          got: '2'
    #     expected: '0'
    # Looks like you failed 1 test of 2.
t/po-files.t .. 1/? 
#   Failed test 'no fuzzy marks'
#   at t/po-files.t line 37.
usage: make [-BeikNnqrSstWwX]
            [-C directory] [-D variable] [-d flags] [-f makefile]
            [-I directory] [-J private] [-j max_jobs] [-m directory] [-T file]
            [-V variable] [-v variable] [variable=value] [target ...]

    #   Failed test 'make check-po exits with value 0'
    #   at t/po-files.t line 41.
    #          got: '2'
    #     expected: '0'
    # Looks like you failed 1 test of 2.

#   Failed test 'check po files'
#   at t/po-files.t line 43.
Can't exec "git": No such file or directory at t/po-files.t line 49.
Use of uninitialized value $output in string ne at t/po-files.t line 51.
usage: make [-BeikNnqrSstWwX]
            [-C directory] [-D variable] [-d flags] [-f makefile]
            [-I directory] [-J private] [-j max_jobs] [-m directory] [-T file]
            [-V variable] [-v variable] [variable=value] [target ...]

    #   Failed test 'make tidy-po exits with value 0'
    #   at t/po-files.t line 54.
    #          got: '2'
    #     expected: '0'
Can't exec "git": No such file or directory at t/po-files.t line 57.

    #   Failed test 'all files are tidied (if not run "make tidy-po")'
    #   at t/po-files.t line 58.
    #          got: undef
    #     expected: ''
    # Looks like you failed 2 tests of 3.

#   Failed test 'tidy po files'
#   at t/po-files.t line 60.
# Looks like you failed 3 tests of 3.
t/po-files.t .. Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/3 subtests 

Test Summary Report
-------------------
t/po-files.t (Wstat: 768 Tests: 3 Failed: 3)
  Failed tests:  1-3
  Non-zero exit status: 3
Files=2, Tests=4,  1 wallclock secs ( 0.01 usr  0.02 sys +  0.62 cusr  0.05 csys =  0.69 CPU)
Result: FAIL
Failed 1/2 test programs. 3/4 subtests failed.
*** Error code 3
```

**Comment by matsduf** [comment]
The PO files are not present in the distribution package.

**Comment by matsduf** [comment]
Resolved by #320
